tests/migration/reboot_to_upgrade.pm

```diff
@@ -36,10 +38,14 @@ sub run {
         set_var('BOOT_HDD_IMAGE', 0) unless (is_aarch64 && !check_var('ZDUP', '1'));
     }
     assert_script_run "sync", 300;
-    enter_cmd "reboot";
+    power_action('reboot', textmode => 1, keepconsole => 1);
```

Comment:
This keep console may not work on s390x, please double check it.

Reply:
https://openqa.nue.suse.com/tests/7087288 PASSED

Reply:
Please run a z/VM case also, https://openqa.nue.suse.com/tests/6995606

Reply:
http://openqa.nue.suse.com/t7088244  PASSED
http://openqa.nue.suse.com/t7088245  PASSED